ui-admin/src/forms/designer/QuestionDesigner.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+
+import { Question } from '@juniper/ui-core'
+
+import { BaseFields } from './questions/BaseFields'
+import { CheckboxFields } from './questions/CheckboxFields'
+import { ChoicesList } from './questions/ChoicesList'
+import { OtherOptionFields } from './questions/OtherOptionFields'
+import { questionTypeDescriptions, questionTypeLabels } from './questions/questionTypes'
+import { VisibilityFields } from './questions/VisibilityFields'
+
+export type QuestionDesignerProps = {
+  readOnly: boolean
+  value: Question
```

Comment:
Maybe  "question" or "targetQuestion" or "item" instead of value?  When I saw 'value' I thought of option values

Reply:
Yeah, I think this began as mimicking a form input interface with value and onChange props. But then I started using question for other stuff as well.

Reply:
Renamed in https://github.com/broadinstitute/juniper/pull/444/commits/531132d544e52ddaed7d13986fac204fb325d63f.